storage/innobase/include/srv0srv.h

```diff
@@ -760,6 +760,9 @@ struct export_var_t{
 	ulint innodb_encryption_rotation_pages_flushed;
 	ulint innodb_encryption_rotation_estimated_iops;
 	int64_t innodb_encryption_key_requests;
+
+	/* Number of bulk insert operations */
+	Atomic_relaxed<ulint> innodb_bulk_operations;
```

Comment:
The comment is misleading, because the counter is also being incremented on some `ALTER TABLE`.

Why do you not use `Atomic_counter` like I suggested? The end result should be the same, though. I also requested to move this counter next to `innodb_instant_alter_column`.